website/docs/usage-training.md

```diff
@@ -293,15 +293,3 @@ values = [0.001, 0.01, 0.1]
 </grid>
 
 ---
-
-## Distributed training {#distributed}
-
-We expect to recommend [Ray](https://ray.io/) for distributed training. Ray
-offers a clean and simple API that fits well with Thinc's model design. While
-full support is still under development, you can find an example here:
-
-<tutorials header="false">
-
-- parallel_training_ray
-
-</tutorials>
```

Comment:
Perhaps we can keep the section and just say "Full support is still under development." and end it at that? We do want to pick this up again some time "soon".

Reply:
That's indeed a better idea; will fix.